ambry-network/src/main/java/com.github.ambry.network/NetworkClient.java

```diff
@@ -178,25 +182,31 @@ public NetworkClient(Selector selector, NetworkConfig networkConfig, NetworkMetr
    *                         the selector events.
    */
   private void handleSelectorEvents(List<ResponseInfo> responseInfoList) {
+    logger.trace("Handling selector events at " + time.milliseconds());
```

Comment:
Same reason as I quoted. This is nothing but slicing the entire time(send and poll) into multiple slices, so that it would help us during investigations.